schema/2.0.0/seurat_encoding.md

```diff
@@ -0,0 +1,226 @@
+
+# Schema - Seurat encoding
+
+Contact: acarr@chanzuckerberg.com
+
+Document Status: _In progress_
+
+Schema version: 2.0.0
+
+
+## Overview
+
+All data submitted to the [cellxgene data portal](https://cellxgene.cziscience.com/) is automatically converted to a Seurat V3 object that can be loaded by the R package [Seurat](https://satijalab.org/seurat/). The conversion is done through [sceasy](https://github.com/cellgeni/sceasy).
+
+This document describes such Seurat encoding for the converted data and it mirrors the data [schema](./schema.md).
+
+## Encoding
+
+The format used to store the Seurat object is [`.rds`](https://stat.ethz.ch/R-manual/R-devel/library/base/html/readRDS.html) which provides an interface for storing individual R objects on disk.
+
+A `local.rds` file downloaded from the cellxgene data portal can be read into an R session by doing the following:
+
+```r
+seurat_object <- readRDS(local.rds)
+```
+
+The following sections describe the individual components of a dataset as encoded in the Seurat object.
+
+### Data matrix
+
+Matrix data is stored in the slot `assays` under the element `RNA`.
+
+#### Raw data
+
+<table><tbody>
+    <tr>
+      <th>Slot</th>
+      <td><code>counts</code></td>
+    </tr>
+    <tr>
+      <th>Access</th>
+      <td><code>seurat_object@assays$RNA@counts</code></td>
+    </tr>
+    <tr>
+      <th>Value</th>
+        <td>A sparse matrix of type <code>dgCMatrix</code></td>
```

Comment:
I'd remove all the "A"'s in Values throughout the document. 

sparse matrix of type ...
data.frame

etc.